thirdeye/thirdeye-frontend/app/pods/rootcause/route.js

```diff
@@ -7,17 +7,77 @@ import { toCurrentUrn, toBaselineUrn, filterPrefix, dateFormatFull } from 'third
 import { checkStatus } from 'thirdeye-frontend/utils/utils';
 import _ from 'lodash';
 
-const queryParamsConfig = {
-  refreshModel: false,
-  replace: false
+const ROOTCAUSE_SETUP_MODE_CONTEXT = "context";
+const ROOTCAUSE_SETUP_MODE_SELECTED = "selected";
+const ROOTCAUSE_SETUP_MODE_NONE = "none";
+
+/**
+ * converts RCA backend granularity strings into units understood by moment.js
+ */
+const toMetricGranularity = (attrGranularity) => {
+  let [count, unit] = attrGranularity.split('_');
+
+  switch (unit) {
+    case 'MINUTES':
+      unit = 'minute';
+      break;
+
+    case 'HOURS':
+      unit = 'hour';
+      break;
+
+    case 'DAYS':
+      unit = 'day';
+      break;
+  }
+
+  return [parseInt(count, 10), unit];
+};
+
+/**
+ * Returns the anomaly time range offset (in granularity units) based on metric granularity
+ */
+const toAnomalyOffset = (granularity) => {
```

Comment:
Harde coded arbitrary nums?

Reply:
yes. if it needs a config, we can package the logic here too